inc/item_devices.class.php

```diff
@@ -105,9 +105,41 @@ function getForbiddenStandardMassiveAction() {
       return $forbidden;
    }
 
-
    function rawSearchOptions() {
-      $tab = parent::rawSearchOptions();
+      $tab = [];
```

Comment:
Is it expected you no longer call parent method?

Reply:
Yes parent provides search options that does not works for this case.

Reply:
OK :)